packages/link/react.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="react" />
+
+export type Appearance = 'default' | 'subtle'
+
+export interface LinkProps {
+  appearance?: Appearance
+}
+
+export default function Link(props: LinkProps): React.Component<LinkProps>
```

Comment:
Is `React.Component` known within this file?  How is this resolved?

Reply:
See comment on the `/// <reference>` line